src/test/interface/metric_test.cpp

```diff
@@ -1,12 +1,12 @@
-#include <stan/services/error_codes.hpp>
+#include <boost/math/policies/error_handling.hpp>
 #include <cmdstan/command.hpp>
 #include <gtest/gtest.h>
+#include <stan/callbacks/stream_writer.hpp>
+#include <stan/services/error_codes.hpp>
+#include <stdexcept>
 #include <string>
-#include <test/utility.hpp>
 #include <test/test-models/proper.hpp>
-#include <stdexcept>
-#include <boost/math/policies/error_handling.hpp>
-#include <stan/callbacks/stream_writer.hpp>
+#include <test/utility.hpp>
```

Comment:
Do we have this header order stuff in Math also?

I personally more prefer the order we have in math:
- stan include
- std:: includes alphabetically.

Reply:
Everything else apart from this looks great!

Reply:
Oh weird I thought this would be respected via the .clang-format. Let me look at this